web/gds-admin-ui/src/__tests__/basic-details.spec.js

```diff
@@ -3,252 +3,158 @@ import userEvent from '@testing-library/user-event'
-                            {
-                                "legal_person_name": "Opalcliff",
-                                "legal_person_name_identifier_type": "LEGAL_PERSON_NAME_TYPE_CODE_SHRT"
-                            }
-                        ],
-                        "name_identifiers": [
-                            {
-                                "legal_person_name": "Opalcliff, Inc.",
-                                "legal_person_name_identifier_type": "LEGAL_PERSON_NAME_TYPE_CODE_LEGL"
-                            }
-                        ],
-                        "phonetic_name_identifiers": []
-                    },
-                    "national_identification": {
-                        "country_of_issue": "US",
-                        "national_identifier": "5",
-                        "national_identifier_type": "NATIONAL_IDENTIFIER_TYPE_CODE_LEIX",
-                        "registration_authority": ""
-                    }
-                },
-                "established_on": "275760-08-08",
-                "extra": null,
-                "first_listed": "2021-07-28T15:44:23Z",
-                "id": "30f624c5-2007-4c2a-8770-31e5087d042d",
-                "identity_certificate": null,
-                "last_updated": "2021-07-28T15:50:06Z",
-                "registered_directory": "trisatest.net",
-                "service_status": "UNKNOWN",
-                "signature": "",
-                "signing_certificates": [],
-                "trisa_endpoint": "opalcliff.com:443",
-                "trixo": {
-                    "applicable_regulations": [
-                        "FATF Recommendation 16"
-                    ],
-                    "compliance_threshold": 0,
-                    "compliance_threshold_currency": "USD",
-                    "conducts_customer_kyc": false,
-                    "financial_transfers_permitted": "",
-                    "has_required_regulatory_program": "",
-                    "kyc_threshold": 0,
-                    "kyc_threshold_currency": "USD",
-                    "must_comply_travel_rule": false,
-                    "must_safeguard_pii": false,
-                    "other_jurisdictions": [],
-                    "primary_national_jurisdiction": "US",
-                    "primary_regulator": "Wife",
-                    "safeguards_pii": false
-                },
-                "vasp_categories": [
-                    "P2P"
-                ],
-                "verification_status": "PENDING_REVIEW",
-                "verified_on": "",
-                "version": {
-                    "pid": "0",
-                    "version": "4"
+    describe("ReviewOption", () => {
+        it("Should not be disable when status is different to pending", async () => {
+
+            const mockVaspData = {
+                "vasp": {
+                    "verification_status": "PENDING_REVIEW",
+                }
+            }
```

Comment:
In the backend we ended up allowing registrations in states <= `PENDING` or in the `ERRORED` state to be deleted. We should make the same change in the UI. In summary, we want to allow deletion in states: `NO_VERIFICATION`, `SUBMITTED`, `EMAIL_VERIFIED`, `PENDING_REVIEW`, and `ERRORED`

and prevent deletion in states: `REVIEWED`, `ISSUING_CERTIFICATE`, `VERIFIED`, `REJECTED`, and `APPEALED`.